container/Dockerfile

```
@@ -1,25 +1,25 @@
-FROM registry.fedoraproject.org/fedora:33
+FROM registry.redhat.io/rhel9/python-39
```

This works if you have a subscription, but this one works even without (for as long as RH maintains this registry):
```suggestion
FROM registry.access.redhat.com/rhel9/python-39
```
shouldn't matter for deployment but will make life just a little easier for developers.